dirsrvtests/tests/suites/clu/dsconf_test.py

```diff
@@ -244,3 +245,28 @@ def test_check_replica_id_accepted_hub_consumer(topology_st, instance_role, repl
     log.info(f'output message : {msg}')
     assert "Replication successfully enabled for" in msg
     log.info(f"Test PASSED: --replica-id option is accepted for {instance_role}.")
+
+
+def test_def_rootdse_attr(topology_st):
+    """Test to check if nsslapd-haproxy-trusted-ip attribute is present in schema by default
+
+    :id: e8aca6d7-3c6f-47e6-ad6c-86fa223ddbac
+    :parameterized: no
+    :setup: Standalone instance
+
+    :steps:
+        1. Run an ldapsearch for nsslapd-haproxy-trusted-ip attribute in default schema
+        2. Check the returned entry
+
+    :expectedresults:
+        1. ldapsearch should return the attribute
+        2. The attribute should be present in default schema
+    """
+    topology_st.standalone.start()
+
+    log.info("Assert default schema has nsslapd-haproxy-trusted-ip attribute")
+    try:
+        haproxy = topology_st.standalone.schema.query_attributetype('nsslapd-haproxy-trusted-ip')
+        assert haproxy[0].names[0] == 'nsslapd-haproxy-trusted-ip'
```

Comment:
I think this check should be added to the existing [test_haproxy_trust_ip_attribute](https://github.com/389ds/389-ds-base/blob/a9c7ff9419a15508071a9ad43e4c7f0469938506/dirsrvtests/tests/suites/basic/haproxy_test.py#L46) test.

Is there a reason to use `assert` in `try` block and then raise an Exception in `except` block? `assert` raises `AssertionError`, I believe it should be enough to fail the test.

`schema.query_attributetype()` returns `None` if the attribute is not found, so we can assert that that it's not `None` instead.

Reply:
Good point. I'll move this check to the above test. Ah yes, AssertionError is enough here. Fixed it